gctscript/examples/csv.gct

```diff
@@ -0,0 +1,28 @@
+exch := import("exchange")
+t := import("times")
+// Import all the indicators you want
+atr := import("indicator/atr")
+sma := import("indicator/sma")
+ema := import("indicator/ema")
+
+load := func() {
+    // define your start and end within reason.
+    start := t.date(2017, 8 , 17, 0 , 0 , 0, 0)
+    end := t.add_date(start, 0, 6 , 0)
+
+    // This fetches the ohlcv 
+    ohlcvData := exch.ohlcv("binance", "BTC-USDT", "-", "SPOT", start, end, "1d")
+
+    // construct ta values 
+    avgtr := atr.calculate(ohlcvData.candles, 14)
+    simma := sma.calculate(ohlcvData.candles, 9)
+    expma := ema.calculate(ohlcvData.candles, 9)
+    
+    // 'output' is already defined when we construct our bytecode from file. 
+    // It points to our default output file in the scripts folder.
+    // Input the rest of the fields, any order, doesn't matter. Once the script
+    // is executed it will save as csv and you can open in excel.
+    common.writeascsv(output, ohlcvData, avgtr, simma, expma)
```

Comment:
I have added in a a condition if the target is "" it will return an error. Don't really have a good idea of handling the multiple strings, I might just set and check again if its already set then error.

Reply:
Yeah, I think only allowing one string to set the name sounds good